src/lib/Refiner.ml

```diff
@@ -17,18 +17,19 @@ open Bwd
 
 type ('a, 'b) quantifier = 'a -> CS.ident option * (T.var -> 'b) -> 'b
 
-let rec int_to_term =
-  function
-  | 0 -> S.Zero
-  | n -> S.Suc (int_to_term (n - 1))
-
 module Hole =
 struct
+  let norm : D.cof -> D.cof m =
+    fun phi ->
+      let* useless = EM.lift_cmp @@ CmpM.test_sequent [phi] Cof.bot in
+      EM.ret (if useless then Cof.bot else phi)
```

Comment:
This looks fine to me! But if want a "Cool" way to write this, you could do:

````ocaml
let+ useless = ... in 
if useless ... 
````

Reply:
ah, gotcha.